src/storage/storage.ts

```diff
@@ -5,8 +5,8 @@ export type File = Buffer | string;
 export type Content = string | Buffer;
 
 export interface StorageInterface {
```

Comment:
Not related, but I wonder if we need another method added to the interface?
```typescript
getUri(targetPath: string): Promise<stringUri>
```
Why? Because it would be needed to be able to build the correct activity pub file.